src/test/getarg_tests.cpp

```diff
@@ -30,6 +30,41 @@ static void ResetArgs(const std::string& strArg)
     gArgs.ParseParameters(vecChar.size(), vecChar.data());
 }
 
+BOOST_AUTO_TEST_CASE(configfile)
+{
+  std::ostringstream args;
+  args << "-conf=" << (fs::path("data") / "bitcoin.conf").string()
+       << " -datadir=" << fs::path("src/test").string();
+  ResetArgs(args.str());
+  gArgs.ReadConfigFile();
+
+  BOOST_CHECK_EQUAL(gArgs.GetConfigFile(), fs::system_complete("test/data/bitcoin.conf"));
```

Comment:
```
test/getarg_tests.cpp(41): error: in 
"getarg_tests/configfile": check gArgs.GetConfigFile() == fs::system_complete("test/data/bitcoin.conf") has failed
["/home/travis/build/bitcoin/bitcoin/build/bitcoin-x86_64-unknown-linux-gnu/src/data/bitcoin.conf" != "/home/travis/build/bitcoin/bitcoin/build/bitcoin-x86_64-unknown-linux-gnu/src/test/data/bitcoin.conf"]

Reply:
Yeah it's running different locally for me than on CI, and have not figured that out yet. In cases where the paths match up as expected (which should mean the config is properly identified), the arg checks do not pass, as though the file is not being picked up.

Reply:
Any idea what's going on here? https://travis-ci.org/bitcoin/bitcoin/jobs/358040567

It's saying the test bitcoin.conf does not exist under CI:

```
test/getarg_tests.cpp(39): error in "configfile": check fs::exists(gArgs.GetConfigFile()) failed
test/getarg_tests.cpp(40): error in "configfile": check gArgs.GetConfigFile() == "test/data/bitcoin.conf" failed ["/home/travis/build/bitcoin/bitcoin/build/bitcoin-x86_64-unknown-linux-gnu/src/test/data/bitcoin.conf" != test/data/bitcoin.conf]
```